src/test/java/org/springframework/integration/aws/config/xml/SqsMessageDrivenChannelAdapterParserTests.java

```diff
@@ -85,6 +86,7 @@ public void testSqsMessageDrivenChannelAdapterParser() {
 		SimpleMessageListenerContainer listenerContainer =
 				TestUtils.getPropertyValue(this.sqsMessageDrivenChannelAdapter, "listenerContainer",
 						SimpleMessageListenerContainer.class);
+		listenerContainer.setAutoStartup(false);
```

Comment:
If I remove this line then the test fails.
`expected:<[fals]e> but was:<[tru]e>`